### PR TITLE
[ISSUE #57] Modify the field, synchronize the modification, and add the mapper method

### PR DESCRIPTION
**Pull request by zzxxiansheng**

### CODE REVIEW

**Comment by Pil0tXia** [comment]
Please resolve conflicts. Besides, please checkout a new branch to develop, otherwise the readability of your commit history would be poor.

**Comment by zzxxiansheng** [comment]
> 请解决冲突。此外，请检查一个新分支进行开发，否则提交历史记录的可读性会很差。

OK

**Comment by Lambert-Rao** [comment]
@Pil0tXia plase review this PR, database related code is in need

File: eventmesh-dashboard-console/src/test/resources/health-test.sql [hunk 1]
```diff
@@ -17,10 +17,10 @@
...
 
-insert into `eventmesh_dashboard_test`.health_check_result (id, type, type_id, cluster_id, status, result_desc, create_time, update_time)
-values  (1, 1, 1, 1, 0, '', '2024-02-02 18:56:50', '2024-02-02 18:56:50'),
-    (2, 2, 2, 1, 1, '', '2024-02-02 18:56:50', '2024-02-02 18:56:50'),
-    (3, 3, 3, 1, 1, '', '2024-02-02 18:56:50', '2024-02-02 18:56:50'),
-    (4, 4, 4, 1, 1, '', '2024-02-02 18:56:50', '2024-02-02 18:56:50'),
-    (5, 1, 2, 1, 1, '', '2024-02-04 18:56:50', '2024-02-02 19:33:13'),
-    (6, 4, 2, 2, 0, '', '2024-02-04 18:56:50', '2024-02-02 19:33:13');
\ No newline at end of file
+insert into `eventmesh_dashboard_test`.health_check_result (id, type, type_id, cluster_id, state, result_desc, create_time, update_time)
```
Comment (Pil0tXia) [inline eventmesh-dashboard-console/src/test/resources/health-test.sql:20]: Why rename status to state?

Comment (zzxxiansheng) [reply eventmesh-dashboard-console/src/test/resources/health-test.sql:20]: `Status` indicates the shutdown or opening of an instance, `state` indicates the state that is unique to some instances (open state), and `is_delete` indicates the deletion or existence of system management data, such as operation record tables.

Comment (Pil0tXia) [reply eventmesh-dashboard-console/src/test/resources/health-test.sql:20]: You may submit changes of comments in the next PR.

**Review by Pil0tXia — APPROVED** [review]

File: eventmesh-dashboard-console/src/test/resources/health-test.sql (hunk 1 above)
Comment (zzxxiansheng) [reply eventmesh-dashboard-console/src/test/resources/health-test.sql:20]: receive

**Pull request merged**